modules/auth_email/scripts/verify_link_email.ts

```diff
@@ -0,0 +1,41 @@
+import { Empty, RuntimeError, ScriptContext } from "../module.gen.ts";
+import { verifyCode } from "../utils/code_management.ts";
+import { IDENTITY_INFO_LINK } from "../utils/provider.ts";
+import { ensureNotAssociatedAll } from "../utils/link_assertions.ts";
+
+export interface Request {
+	verificationId: string;
```

Comment:
afaik we can't assume uuids are secure. we should probably issue a random token that we compare using the crypto module. i don't want to use the tokens module for this since that goes to the db and that's overkill. we probably want to expose a utility func from `tokens` that generates a token string, without creating a row etc.

Reply:
Done!